app/models/customer.rb

```diff
@@ -1,17 +1,19 @@
 class Customer < ApplicationRecord
   include PgSearch::Model
-  pg_search_scope :stemmed, ->(query, threshold) {
+  pg_search_scope :stemmed, lambda { |query, threshold|
```

Comment:
Why switch to the older, longer form? FWIW, I use the short form because that's what `standardrb` prefers - and as soon as I run `standardrb`, it's going to change it back.